metricbeat/module/ceph/health/health_test.go

```diff
@@ -0,0 +1,89 @@
+package health
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"path/filepath"
+	"testing"
+
+	"github.com/elastic/beats/libbeat/common"
+	mbtest "github.com/elastic/beats/metricbeat/mb/testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFetchEventContents(t *testing.T) {
+	absPath, err := filepath.Abs("./testdata/")
+
+	response, err := ioutil.ReadFile(absPath + "/sample_response.json")
```

Comment:
This is a great way to test the fetcher with predefined data. We should apply the same logic to other metricsets.